### DISCUSSION TITLE
Files qfields cloud in smartphone

### DISCUSSION THREAD

**Discussion by apprenti1973** [opengisch/QField#3425] [Q&A]
Hello, can someone tell me where I can find the qfieldcloud files stored on a smartphone (offline entry)

**Comment by suricactus** [comment]
If you open the project file, then click top-left menu button, you can find the "Open project folder" button there.

![image](https://user-images.githubusercontent.com/2820439/194839827-c3feec8a-0033-43c6-81ae-5d8ab56ff6ce.png)

**Comment by apprenti1973** [comment]
Thank you
